### PR TITLE
preparing for 1.8

### PR DESCRIPTION
**Pull request by oetiker**

### CODE REVIEW

**Comment by oetiker** [comment]
@c72578 it seems the Windows MSVC build is failing ... any ideas ?

**Comment by c72578** [comment]
> @c72578 it seems the Windows MSVC build is failing ... any ideas ?

I will prepare a fix.

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+--------
+* python bindings: properly convert double values of rrd info <Stefan Gluszek>
+* failed to expand 'Py_UNUSED', Invalid usage when expanding  'Py_UNUSED' <c72578>
+* docment --showtime in xport help output <c72578>
+* fix --use-nan-for-all-missing-data <c72578>
+* update rrdruby.pod <c72578>
+* add missing rrdruby.pod and rrdpython.pod to dist <c72578>
+* Set first_weekday to 0 (Sunday), when HAVE__NL_TIME_WEEK_1STDAY is not defined <nirgal>
+* fix median calculation for all NaN inputs <c72578>
+* fix potetial leak in xport during failure <c72578>
```
Comment (c72578) [inline CHANGES:15]: potetial -> potential

File: CHANGES [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+* fix many warings raised by Cppcheck <c72578>
```
Comment (c72578) [inline CHANGES:16]: warings -> warnings

File: CHANGES [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+* for SMIN example in docs <Thomas-Gelf>
+* fix for pyton3 compatiblity <SuperHeron>
+* freemem only for valid status <Christian Kr"oger>
+* fix double meaning of time 0 as uninitialized value <neirbowj>
+* fix for zfs not supporting fallocate. this makes resize work on zfs <zelsing>
+* add rrdrados.pod to dist <Jeron Roovers>
+* fetch - do not call rrd_freemem on uninitialized pointers <commx>
+* use separate pango fontmap per thread <nomis>
+* switch to python 3 <nirgal>
+* do not leak filename when opeing a broken file <mistotebe>
```
Comment (c72578) [inline CHANGES:34]: opeing -> opening

File: CHANGES [hunk 4]
```diff
@@ -1,3 +1,58 @@
+
+RRDtool 1.8.0 - 2022-03-13
+==========================
+
+Bugfixes
+--------
+* python bindings: properly convert double values of rrd info <Stefan Gluszek>
+* failed to expand 'Py_UNUSED', Invalid usage when expanding  'Py_UNUSED' <c72578>
+* docment --showtime in xport help output <c72578>
```
Comment (c72578) [inline CHANGES:9]: docment -> document

File: CHANGES [hunk 5]
```diff
@@ -1,3 +1,58 @@
...
+* fix many compiler warnings from latest gcc <c72578>
+* ensure proper initialization in rrd_daemon <clarfonthey>
+* cleanup testsuite <nirgal>
+* better testing <c72578>
+* avoid invalid read in rrd_client <pallas>
+* add symbols from rrdc to librrd <pallas>
+* Fix duplicate write_changes_to_disk() calls when HAVE_LIBRADOS is true and HAVE_MMAP is false <yil-cpr>
+* documentation updates <nirgal>
+* for SMIN example in docs <Thomas-Gelf>
+* fix for pyton3 compatiblity <SuperHeron>
```
Comment (c72578) [inline CHANGES:26]: compatiblity -> compatibility

**Review by c72578 — COMMENTED** [review]
Some typos in CHANGES

**Comment by c72578** [comment]
Concerning the failing Linux Build, e.g. [1]:
```console
running egg_info
error: [Errno 13] Permission denied
```
In this commit, `sudo chown -R travis rrdtool.egg-info` has been added to .travis.yml
https://github.com/oetiker/rrdtool-1.x/commit/25491f062ae2104663c19e9ad843c99e1e81f00f

[1] https://github.com/oetiker/rrdtool-1.x/runs/5533122181

**Pull request merged**